packages/super-components/components/msc-billing-tile/src/msc-billing-tile/menu-custom.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component, h, State, Host } from '@stencil/core';
+import { OdsThemeColorIntent } from '@ovhcloud/ods-theming';
+import { OdsButtonVariant, OdsButtonType } from '@ovhcloud/ods-core';
+
+export interface IMenuCustom {
+  showTooltip: boolean;
+}
+
+@Component({
+  tag: 'menu-custom',
+  styleUrl: 'menu-custom.scss',
+  shadow: true,
+})
+export class MenuCustom implements IMenuCustom {
+  @State() showTooltip = false;
+
+  private toggleTooltip(event: Event) {
+    event.stopPropagation();
+    this.showTooltip = !this.showTooltip;
+  }
+
+  render() {
+    return (
+      <Host>
+        <div class="menu-button menu-button-renew">
```

Comment:
I don't see where menu-button-renew is defined

Reply:
Removed ✅